pywwise/ak/wwise/core/transport.py

```diff
@@ -1,54 +1,154 @@
+        :arg game_object: The game object to use for playback.
+        :return: Transport object ID to be used with all other ak.wwise.core.transport functions. Returns -1 if function
+        call failed.
+        """
+        if wwise_object is None:
+            return -1
+        
+        args = {"object": wwise_object if not
+                isinstance(wwise_object, tuple) else f"{wwise_object[0]}_{wwise_object[1]}"}
+        
+        if game_object is not None:
+            args["gameObject"] = game_object
+        
+        result = self._client.call("ak.wwise.core.transport.create", args)
+        
+        return -1 if result is None else result.get("transport", -1)
+    
+    def destroy(self, transport_id: int) -> bool:
+        """
+        https://www.audiokinetic.com/en/library/edge/?source=SDK&id=ak_wwise_core_transport_destroy.html \n
+        Destroys the given transport object.
+        :param transport_id: The transport object ID to be used with all other ak.wwise.core.transport functions. Make
+        sure you create a transport object before calling this function.
+        :return: True if the transport object was destroyed, False otherwise.
+        """
+        if transport_id is None:
+            return False
+        
+        args = {"transport": transport_id}
+        
+        return self._client.call("ak.wwise.core.transport.destroy") is not None
+    
+    def execute_action(self, transport_action: ETransportExecuteActions, transport_id: int = None) -> bool:
+        """
+        https://www.audiokinetic.com/en/library/edge/?source=SDK&id=ak_wwise_core_transport_executeaction.html \n
+        Executes an action on the given transport object, or all transport objects if none is specified.
+        :arg transport_action: The action to execute. Use ETransportExecuteActions to see the available actions.
+        :arg transport_id: The transport object ID to be used with all other ak.wwise.core.transport functions. Make
+        sure you create a transport object before calling this function.
+        :return: True if the action was executed, False otherwise.
+        """
+        if transport_action is None:
+            return False
+        
+        args = {"action": transport_action, **({"transport": transport_id} if transport_id is not None else {})}
+        
+        return self._client.call("ak.wwise.core.transport.execute", args) is not None
+
+    def get_list(self) -> tuple[TransportObjectInfo, ...]:
+        """
+        https://www.audiokinetic.com/en/library/edge/?source=SDK&id=ak_wwise_core_transport_getlist.html \n
+        Returns the list of transport objects.
+        :return: A list of transport objects.
+        """
+        results = self._client.call("ak.wwise.core.transport.getList", {})
+        
+        results = results.get("list") if results is not None else None
+        
+        if results is None:
+            return ()
+        
+        transport_objects: list[TransportObjectInfo]
```

Comment:
At the end, append ` = []` for an explicit initialization. If the typecheck in PyCharm complains, then do `transport_objects = list[TransportObjectInfo]()`.